packages/blockchain/test/blockValidation.spec.ts

```diff
@@ -390,7 +390,7 @@ describe('[Blockchain]: Block validation tests', () => {
   })
 })
 describe('EIP 7685: requests field validation tests', () => {
-  it('should throw when putting a block with an invalid requestsRoot', async () => {
+  it('should throw when putting a block with an invalid requestsHash', async () => {
```

Comment:
This test should be deleted, it is now not possible (in blockchain) to verify the correctness in `requestsHash`. We need to run the block in VM to generate the requests, and then verify that if we hash it according to the algorithm it yields the same hash.